src/main/webapp/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Black Student Insight</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload="hideElements()">
+    <header>
+      <nav>
+        <ul>
+          <li><a href="index.html">Black Student Insight</a></li> 
```

Comment:
It seems like this doesn't have a function because it is linking to the page it's already on. Instead, you can add the header that is already on the college-info.html and comments.html pages.

Reply:
I made it serve as a home button. I forgot to make the change in the comments page, but if you look at the college-info.html page, you'll see that onclick it'll redirect the client to the home page. I thought that would be more convenient than the back to home link since I wanted to add more pages in the future.